content/client-lib-development-guide/features.textile

```diff
@@ -368,9 +368,8 @@ h3(#realtime-connection). Connection
 ** @(RTN9a)@ Is @null@ until connected
 ** @(RTN9b)@ Is a unique private connection key provided by Ably that is used to reconnect and retain connection state following an unexpected disconnection. You should have a test to ensure multiple connected clients have unique connection keys
 * @(RTN10)@ @Connection#serial@ attribute:
-** @(RTN10a)@ Is set to -1 once @CONNECTED@
-** @(RTN10b)@ Is always 0 when the first message has been sent and an @ACK@ has been received confirming receipt of the message. The @serial@ is updated whenever a @ProtocolMessage@ is received that contains a @connectionSerial@ value. A test should exist that checks that the serial number is not updated when a message is sent, but then increments by one when the @ACK@ is received
-** @(RTN10c)@ The last known @ProtocolMessage@ @connectionSerial@ is used when restoring connection state
+** @(RTN10a)@ Is @null@ until connected, and is then set from the @connectionSerial@ attribute of the @CONNECTED@ @ProtocolMessage@
+** @(RTN10b)@ Continues to be updated by every @ProtocolMessage@ received from Ably that contains a @connectionSerial@. A test should exist that checks that the serial is updated when a message is received with the value from that message. It should not otherwise change; in particular, unlike the library-internal @msgSerial@, it should not change as a result of publishing a message or receiving an @ACK@.
```

Comment:
>  when a message is received with the value from that message

Could we be more explicit and specify a `@ProtocolMessage@` instead of `a message`?